packages/api/src/controllers/stripe.ts

```diff
@@ -375,34 +379,47 @@ app.post("/webhook", async (req, res) => {
   return res.sendStatus(200);
 });
 
-app.post("/subscribe-hackers-to-pay-as-you-go", async (req, res) => {
+app.post("/hacker/migration/pay-as-you-go", async (req, res) => {
   if (req.config.stripeSecretKey != req.body.stripeSecretKey) {
     res.status(403);
     return res.json({ errors: ["unauthorized"] });
   }
   let migration = [];
 
-  const [users] = await db.user.find(
-    [
-      sql`users.data->>'stripeProductId' = 'prod_O9XuIjn7EqYRVW' 
-          AND (users.data->>'toMigrate' = 'true' OR users.data->>'toMigrate' IS NULL)`,
-    ],
-    {
-      limit: 1,
-      useReplica: false,
-    }
-  );
+  let hackerPlan = "prod_O9XuIjn7EqYRVW";
 
-  if (users.length == 0) {
-    res.json({
-      errors: ["no users found"],
-    });
-    return;
+  if (req.body.staging === true) {
```

Comment:
You should probably check something in the config instead of allowing prod to execute staging logic and staging to execute prod logic depending on a request parameter. e.g.
```suggestion
  if (req.config.frontendDomain === 'livepeer.monster') {
```
https://github.com/livepeer/livepeer-infra/blob/66a29742e156bc65887b5ce70941bb1d785a5eb0/values/api.values.yaml#L40

Ideally all these product IDs would come from config and/or DB, as we've already talked though :)